addons/web/static/src/js/fields/basic_fields.js

```diff
@@ -1941,7 +1940,10 @@ var FieldBinaryFile = AbstractFieldBinary.extend({
                     'data': utils.is_bin_size(this.value) ? null : this.value,
                 },
                 'complete': framework.unblockUI,
-                'error': c.rpc_error.bind(c),
+                'error': function () {
```

Comment:
- no `'`
- can use ES6 arrow function, that way you don't need `var self = this;`:

```js
error: () => this.call('crash_manager', 'rpc_error', ...arguments),
```